db/seeds.rb

```diff
@@ -14,3 +15,15 @@
 Language.find_or_create_by({id:2,name:'Español',code:'ES'})
 Language.connection.execute("select setval('languages_id_seq', (select max(id) from languages))")
 
+
+Currency.find_or_create_by({id:1,name:'Argentinean Peso',code:'ARS',symbol:'AR$'})
+Currency.find_or_create_by({id:2,name:'U.S. Dollar',code:'USD',symbol:'US$'})
+Currency.find_or_create_by({id:3,name:'British Pound',code:'GBP',symbol:'£'})
```

Comment:
I know it wasn't so in the old db, but we should be consistent with the symbols.